src/MessageInterface.php

```diff
@@ -25,10 +26,8 @@
  * ;
  * $mailer->send($message);
  * ```
- *
- * @psalm-type FromType = array<string, string>|string
  */
-interface MessageInterface
+interface MessageInterface extends Stringable
```

Comment:
It would be good to describe what's expected from `__toString()` here. A full rendered message or debug info or something else.

Reply:
done